tests/test-recipes/metadata/entry_points/meta.yaml

```diff
@@ -20,3 +20,7 @@ requirements:
     - setuptools
   run:
     - python
+
+# Ensure we get different build strings for concurrently tested packages.
+extra:
+  dummy: '{{ pytest_name is defined }}'
```

Comment:
I don't get this part 🤔 Isn't this going to always be true? And how are these `extra` key-value pairs affecting build strings? I thought only `conda_build_config` keys got there, as well as some host deps?

Reply:
`conda-build` computes the hash from the set of used variant variables
To determine which variables are used, it searches the recipes for occurrences.
For `meta.yaml` this means if it finds a `{{ this_variable<whatever> }}`, it adds `this_variable` to the set of variables.
The hash in the build string will thus change depending on the value of `this_variable` (totally ignoring how it is otherwise used in the recipe).

Reply:
> I thought only `conda_build_config` keys got there, as well as some host deps?

Those are only the inputs (which I provide in the test functions via `variants={"pytest": [request.node.name]}`) but what actually makes it into the hash is determined as written above.